### PR TITLE
feat: add Jenkins pipeline and url templates to provider and project

### PR DESCRIPTION
**Pull request by malta895**

### CODE REVIEW

File: packages/console-types/src/types/provider.ts [hunk 1]
```diff
@@ -77,10 +76,27 @@ const gitProviderCapabilitySchema = {
...
+    name: { const: CI_CD_TOOL },
+    functionalities: providerFunctionalitiesSchema,
+    projectDeployPipelineNameTemplate: { type: 'string' },
+    servicesPipelineNameTemplate: { type: 'string' },
+    apiBaseUrlPathTemplate: { type: 'string' },
+  },
+} as const
+
 export const providerCapabilitySchema = {
-  if: isGitProviderCapability,
```
Comment (mattia-fumo) [inline packages/console-types/src/types/provider.ts:81]: is `isGitProviderCapability` still used anywhere else?

Comment (malta895) [reply packages/console-types/src/types/provider.ts:81]: yes, in [provider types](https://github.com/mia-platform/console-sdk/blob/f78d9c6cb097111b9dd4073ad628f6c26568e489/packages/console-types/src/types/providerType.ts#L94)

Comment (malta895) [reply packages/console-types/src/types/provider.ts:81]: we actually could remove the export from there, is not needed anymore

Comment (malta895) [reply packages/console-types/src/types/provider.ts:81]: done

**Review by mattia-fumo — DISMISSED** [review]
LGTM

**Pull request merged**